templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -58,10 +58,10 @@
 						</div>
 						<div class="gt-df gt-ac gt-gap-3">
 							{{if (and $.Permission.IsAdmin (or (eq .Review.Type 1) (eq .Review.Type 3)) (not $.Issue.IsClosed))}}
-								<a href="#" class="ui muted icon gt-df gt-ac dismiss-review-btn" data-review-id="dismiss-review-{{.Review.ID}}" data-tooltip-content="{{$.locale.Tr "repo.issues.dismiss_review"}}">
+								<a href="#" class="ui muted icon gt-df gt-ac show-modal" data-review-id="dismiss-review-{{.Review.ID}}" data-tooltip-content="{{$.locale.Tr "repo.issues.dismiss_review"}}" data-modal="#dismiss-review-modal">
 									{{svg "octicon-x" 20}}
 								</a>
-								<div class="ui small modal" id="dismiss-review-modal">
+								<div class="ui small modal dismiss-review-modal" id="dismiss-review-modal">
```

Comment:
This PR is wrong.

You are using the same `id="dismiss-review-modal"` in a `range` loop

Please revert or fix ASAP.

Reply:
The id is not introduced by this PR so we can send a new PR to fix it.

Reply:
This ID was never used.

But this PR uses it by `data-modal="#dismiss-review-modal"`.

Then which `modal` does it refer to?

Reply:
The problem is inside `range` there will be multiple iterations but HTML ids must be unique in the whole page. I think the solution is to move the modal outside the `range` as there can effectively only be one visible at a time.

Reply:
Yes, seems like a good temporary solution. The modal depends on `.Review.ID` so can not be easily extracted outside the loop as-is.